changelogs/unreleased/gh-8967-creds-postope-sync-when-RO-till-RW.md

```diff
@@ -0,0 +1,5 @@
+## feature/config
+
+* On read-only instances, Tarantool now synchronizes credentials
+  in the background when switching to read-write mode instead of
+  skipping the synchronization (gh-8967).
```

Comment:
711756d727456bddd39ee52945dd5ae27a1a5684 is in `master`. It worth to rebase.

Reply:
Done.